### PR TITLE
Added operation duration to the finishing log.

### PR DESCRIPTION
**Pull request by uKL**

### CODE REVIEW

**Comment by uKL** [comment]
@dariuszseweryn, refactored a bit.

File: rxandroidble/src/main/java/com/polidea/rxandroidble/internal/util/OperationLogger.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package com.polidea.rxandroidble.internal.util;
+
+import android.support.annotation.RestrictTo;
+
+import com.polidea.rxandroidble.internal.RxBleLog;
+import com.polidea.rxandroidble.internal.operations.Operation;
+
+public class OperationLogger {
+
+    @RestrictTo(RestrictTo.Scope.LIBRARY)
```
Comment (dariuszseweryn) [inline rxandroidble/src/main/java/com/polidea/rxandroidble/internal/util/OperationLogger.java]: Not really needed since it is under `internal` package

**Comment by uKL** [comment]
Let me check the delegation, you may be right.

**Comment by uKL** [comment]
Actually no, since it's a static method it works.

```
01-16 12:19:59.817 19756-19756/com.polidea.rxandroidble.sample D/RxBle#ClientOperationQueue: QUEUED ScanOperationApi21(123696767)
01-16 12:19:59.818 19756-19786/com.polidea.rxandroidble.sample D/RxBle#ClientOperationQueue: STARTED ScanOperationApi21(123696767)
01-16 12:19:59.820 19756-20062/com.polidea.rxandroidble.sample I/RxBle#QueueOperation: Scan operation is requested to start.
01-16 12:19:59.933 19756-19786/com.polidea.rxandroidble.sample D/RxBle#ClientOperationQueue: FINISHED in 115ms ScanOperationApi21(123696767)
01-16 12:20:07.645 21313-21313/com.polidea.rxandroidble.sample D/RxBle#ClientOperationQueue: QUEUED ScanOperationApi21(64820451)
01-16 12:20:07.646 21313-21328/com.polidea.rxandroidble.sample D/RxBle#ClientOperationQueue: STARTED ScanOperationApi21(64820451)
01-16 12:20:07.649 21313-21349/com.polidea.rxandroidble.sample I/RxBle#QueueOperation: Scan operation is requested to start.
01-16 12:20:07.767 21313-21328/com.polidea.rxandroidble.sample D/RxBle#ClientOperationQueue: FINISHED in 121ms ScanOperationApi21(64820451)
01-16 12:20:08.700 21313-21349/com.polidea.rxandroidble.sample I/RxBle#CancellableSubscription: Scan operation is requested to stop.
```

**Comment by uKL** [comment]
Is this what you meant?

```
01-16 12:37:17.492 24123-24123/com.polidea.rxandroidble.sample D/RxBle#ClientOperationQueue: QUEUED   ScanOperationApi21(39162453)
01-16 12:37:17.494 24123-24151/com.polidea.rxandroidble.sample D/RxBle#ClientOperationQueue: STARTED  ScanOperationApi21(39162453)
01-16 12:37:17.503 24123-24416/com.polidea.rxandroidble.sample I/RxBle#QueueOperation: Scan operation is requested to start.
01-16 12:37:17.623 24123-24151/com.polidea.rxandroidble.sample D/RxBle#ClientOperationQueue: FINISHED ScanOperationApi21(39162453) in 131 ms
01-16 12:37:20.843 24123-24416/com.polidea.rxandroidble.sample I/RxBle#CancellableSubscription: Scan operation is requested to stop.
01-16 12:37:23.275 24123-25202/com.polidea.rxandroidble.sample D/RxBle#ClientOperationQueue: QUEUED   ConnectOperation(101216245)
01-16 12:37:23.276 24123-24151/com.polidea.rxandroidble.sample D/RxBle#ClientOperationQueue: STARTED  ConnectOperation(101216245)
01-16 12:37:23.282 24123-24416/com.polidea.rxandroidble.sample V/RxBle#BleConnectionCompat: Connecting without reflection
01-16 12:37:23.532 24123-24136/com.polidea.rxandroidble.sample D/RxBle#BluetoothGatt: onConnectionStateChange newState=2 status=0
01-16 12:37:23.534 24123-24151/com.polidea.rxandroidble.sample D/RxBle#ClientOperationQueue: FINISHED ConnectOperation(101216245) in 259 ms
```

File: rxandroidble/src/main/java/com/polidea/rxandroidble/internal/serialization/ClientOperationQueueImpl.java [hunk 2]
```diff
@@ -1,9 +1,11 @@
 package com.polidea.rxandroidble.internal.serialization;
 
 import android.support.annotation.RestrictTo;
+
```
Comment (dariuszseweryn) [inline rxandroidble/src/main/java/com/polidea/rxandroidble/internal/serialization/ClientOperationQueueImpl.java]: We should include the formatting configuration into the repository so these kind of changes would not happen.

**Pull request merged**